tests/test_dt.py

```diff
@@ -32,9 +32,15 @@
 from collections import namedtuple
 from datatable import stype, ltype
 from datatable.internal import frame_column_rowindex, frame_integrity_check
+from datatable.lib import core
 from tests import same_iterables, list_equals, noop, isview, assert_equals
 
 
+#-------------------------------------------------------------------------------
+# Check if we need to run C++ tests
+#-------------------------------------------------------------------------------
+
+cpp_tests_enabled = hasattr(core, "test_coverage")
```

Comment:
There is a pytest fixture for this kind of functionality: https://docs.pytest.org/en/latest/skipping.html
```
cpp_test = pytest.mark.skipif(not hasattr(core, "test_coverage"), reason="C++ tests were not compiled")
```
and then mark each test with `@cpp_test`.

Reply:
Sure, let me change it in this way.

Reply:
Done.